src/analysis/normed_space/basic.lean

```diff
@@ -334,6 +346,18 @@ have tendsto f a (𝓝 0) ↔ tendsto (λ e, ∥ f e - 0 ∥) a (𝓝 0) :=
   tendsto_iff_norm_tendsto_zero,
 by simpa
 
+lemma squeeze_zero_norm' {f : γ → α} {g : γ → ℝ} {t₀ : filter γ}
```

Comment:
We should come up with some naming convention for `filter` variables. Unfortunately, I have no idea what would be a good convention.